lib/manageiq/providers/amazon/engine.rb

```diff
@@ -1,4 +1,4 @@
-require 'rails/all'
+require 'action_controller/railtie' # needed for isolate_namespace below
```

Comment:
@jrafanie ??? `isolate_namespace` isn't ActionController; you're just getting it as a side effect.

This should be `rails/engine`, though I note you already mention that above sorta. Huh? I don't get the reasoning for not doing that.